decree-cgc-cfe/fs/binfmt_cgc.c

```diff
@@ -528,21 +560,17 @@ static int load_cgcos_binary(struct linux_binprm *bprm)
 	regs->fs = __USER_DS;
 	ret = 0;
 out:
-	if (phdrs)
-		kfree(phdrs);
-
-	if (s2e_decree_monitor_enabled) {
-		s2e_printf("binfmt_cgc: detected process load %s ret=%d\n",
-			   bprm->interp, ret);
-	}
-
 	if (ret == 0 && s2e_decree_monitor_enabled) {
-		s2e_decree_process_load(current->pid, current->comm, current,
-					&hdr, sizeof(hdr), bprm->interp,
-					hdr.c_entry);
+		s2e_decree_module_load(bprm->interp, current->pid, hdr.c_entry,
+				       elf_phdr, elf_phdr_size);
```

Comment:
Weird mix of tabs and spaces here. What's the rationale for this formatting? Maybe add more detail to the commit message.

Reply:
Ooooh wait, never mind. I just realised that I was in the kernel repo. Ignore me.